File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RoadRunner/messages/TankCommandMessage.java
package org.firstinspires.ftc.teamcode.RoadRunner.messages;

public final class TankCommandMessage {
    public long timestamp;
    public double voltage;
    public double leftPower;
    public double rightPower;

    public TankCommandMessage(double voltage, double leftPower, double rightPower) {
        this.timestamp = System.nanoTime();
        this.voltage = voltage;
        this.leftPower = leftPower;
        this.rightPower = rightPower;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drivetrain/messages/TankCommandMessage.kt
package org.firstinspires.ftc.teamcode.drivetrain.messages

class TankCommandMessage(voltage: Double, leftPower: Double, rightPower: Double) {
    var timestamp: Long
    var voltage: Double
    var leftPower: Double
    var rightPower: Double

    init {
        this.timestamp = System.nanoTime()
        this.voltage = voltage
        this.leftPower = leftPower
        this.rightPower = rightPower
    }
}
